package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "Dictionary Completion",
     "description": "Word Completion",
     "icon": "images/dictionary.png",
-    "version": "1.2.2",
+    "version": "1.2.3",
```

Comment:
Can you revert the changes to `package.json` and `package-lock.json`? I usually keep them in a separate commit.

Reply:
Done.